mantidproject/mantid
# New D11B and D22B SANS machines at ILL

**Description of work.**
Defines 2 new PSD-based multi-panel SANS detectors D11B and D22B.
Also modifies the definition of D33 for tube based masking.
This change however does not alter the detector IDs and detector to count mapping.



**To test:**

1.  Load the new IDFs with LoadEmptyInstrument. 
2. Check that for all of the instruments, in each of the panel the detector IDs start from bottom left corner (looking from the sample) and increment first upwards, then rightwards.
3. Make sure that with the new tube masker tool in Draw tab tubes get masked as follows:
- vertically for
 >  left and right panels for D11B
 >  both panels of D22B
 >  left and right panels for D33
- horizontally for
 > central panel of D11B
 > central, top and bottom panels of D33

**Reviewer, merge with the corresponding generators in mantidgeometry: PR [#176](https://github.com/mantidproject/mantidgeometry/pull/176)**


Part of #26906 . 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
